flink-table/flink-table-planner-blink/src/main/java/org/apache/flink/table/api/PlannerConfigOptions.java

```diff
@@ -39,102 +41,134 @@
 							"(only count RexCall node, including leaves and interior nodes). Negative number to" +
 							" use the default threshold: double of number of nodes.");
 
+	@Documentation.TableOption(execMode = Documentation.ExecMode.BOTH)
```

Comment:
since BOTH is the default you could omit the explicit execMode. Otherwise, please remove the default.

Reply:
Thanks, I have remove the default.